packages/blockchain-link-utils/src/blockbook.ts

```diff
@@ -1,3 +1,7 @@
+// @ts-expect-error
```

Comment:
should not be needed anymore, I added declaration file. Maybe it has to be moved from `packages/suite` somewhere more global, not sure

`everstake.d.ts`

(you have to rebase to current develop)